src/main/scala/ch/openolitor/stammdaten/StammdatenDBEventEntityListener.scala

```diff
@@ -558,17 +558,17 @@ class StammdatenDBEventEntityListener(override val sysConfig: SystemConfig) exte
 
       stammdatenUpdateRepository.getAboDetailAusstehend(abw.aboId) match {
         case Some(abo) => {
-          stammdatenUpdateRepository.getKorb(abw.lieferungId, abw.aboId).toList ++
-            stammdatenUpdateRepository.getZusatzAboKorb(abw.lieferungId, abw.aboId) map { korb =>
-              stammdatenUpdateRepository.getAbotypById(abo.abotypId) map { abotyp =>
-                //re count because the might be another abwesenheit for the same date
-                val newAbwesenheitCount = stammdatenUpdateRepository.countAbwesend(abw.aboId, abw.datum)
-                val status = calculateKorbStatus(newAbwesenheitCount, abo.guthaben, abotyp.guthabenMindestbestand)
+          stammdatenUpdateRepository.getAbotypDetail(abo.abotypId) map { abotyp =>
```

Comment:
I thought we moved the lieferplanung and korb handling to the level 0 operations?